### PR TITLE
initial solution for looping through results

### PR DESCRIPTION
**Pull request by esilgard**
the total number of returned results still aren't matching what I see online, but baby steps

### CODE REVIEW

File: yelp_query.py [hunk 1]
```diff
@@ -25,27 +29,38 @@ def query_categories(key, category_alias):
...
-	print('The status code is {}'.format(req.status_code))
-	if req.ok:
-		output_results(req)
-	else:		
-		print(req.raise_for_status())
\ No newline at end of file
+	max_query_results = np.inf
+	offset = 0
+	results_list = []
+	while offset < max_query_results:
```
Comment (WhiteAu) [inline yelp_query.py:50]: does this ever output? as far as I can tell offset < infinity should continue forever. Maybe a AND statement to catch the end state?

File: yelp_query.py [hunk 2]
```diff
@@ -25,27 +29,38 @@ def query_categories(key, category_alias):
...
+		print('The status code is {}'.format(req.status_code))
+		if req.ok:
+			results = json.loads(req.text)
+			max_query_results = results['total']
+			results_list.append(results['businesses'])
+			num_results = len(results['businesses'])
+			print ('{} returned points of interest'.format(num_results))
+			print ('total response results: {}'.format(max_query_results))
+
+			offset += num_results
```
Comment (WhiteAu) [inline yelp_query.py:62]: if num_results != max_query_results I *think* you should be at the end state...

**Review by WhiteAu — APPROVED** [review]
lgtm. one quick comment below.

File: yelp_query.py (hunk 1 above)
Comment (esilgard) [reply yelp_query.py:50]: yup, it ends (at 52 for this query) the total is returned with each query so it gets reset immediately when returned. although I've got to fix up breaking the loop when there's no return

**Pull request merged**